### ISSUE TITLE
backend style for float_left and float_right

### ISSUE THREAD

**Issue by fritzmg** [contao/core#8239]
With Contao `3.3.0` the inline style `float:left;`/`float:right;` for the `<figure>` were removed from the `ce_text` and `ce_image` templates.

However, this also affected the backend view - images of `text` elements no longer float left or right of the text, which was a bit confusing for some editors, who were used to see this in the backend as well.

I'd suggest to simply add a backend style like
```
.image_container.float_left { float:left; }
.image_container.float_right { float:right; }
```
or something similar as well.

**Comment by Samson1964** [comment]
Seit meinem Update von 3.2 auf 3.5.6 floatet bei mir im Backend auch nichts mehr.

**Comment by leofeyer** [comment]
Fixed in 544d1d35a6f8b8522419ea756847447e3c6d36b9.
